src/test/e2e/25_helm_test.go

```diff
@@ -64,6 +64,10 @@ func testHelmChartsExample(t *testing.T) {
 	defer e2e.CleanFiles(t, fmt.Sprintf("zarf-package-helm-charts-local-tgz-%s-0.0.1.tar.zst", e2e.Arch))
 
 	// Create a package that needs dependencies
+	stdOut, stdErr, err = e2e.Zarf(t, "tools", "helm", "repo", "add", "gitlab", "https://charts.gitlab.io/")
```

Comment:
I imagine the tests previously assumed someone already had this installed, in that case this is a good change